src/summary/documentation/dimensions-card-level-one.css

```diff
@@ -0,0 +1,43 @@
+.table {
```

Comment:
@joeldenning these css classes have been repeated 3 times in different widgets to render data in a responsive table, do you think we should abstract this now?